R/check_exe.R

```diff
@@ -0,0 +1,83 @@
+#' Find location of executable within path or specified directory
+#'
+#' Check that the executable name provided in `model`,
```

Comment:
I think this documentation may need to be updated, as I think instances of `model` have been changed to `exe` in this PR?

May also want to make it clear that this is an "ss3 executable" and not just any old executable.

Reply:
Done.